cql3/expr/expression.cc

```diff
@@ -2648,5 +2648,25 @@ bool is_partition_token_for_schema(const expression& maybe_token, const schema&
 
     return is_partition_token_for_schema(*fun_call, table_schema);
 }
+
+void
+verify_no_aggregate_functions(const expression& expr, std::string_view context_for_errors) {
+    auto find_agg = overloaded_functor{
+            [] (const functions::function_name& f) -> bool {
+                on_internal_error(expr_logger, "verify_no_aggregate_functions: unprepared function_call");
+            },
+            [] (const shared_ptr<functions::function> f) -> bool {
+                return f->is_aggregate();
+            }
+        };
+    bool found_agg = find_in_expression<function_call>(expr, [find_agg] (const function_call& fc) {
+        return std::visit(find_agg, fc.func);
+    });
+    if (found_agg) {
+        throw exceptions::invalid_request_exception(fmt::format("Aggregation function are not supported in the {}", context_for_errors));
```

Comment:
It's because I don't want the 20-patch series to fail because some dtest somewhere checks for this message.  Grammar fixes can be done in independent patches elsewhere.

Reply:
This is true, but a day after you send this PR, nobody will ever remember to fix the message string :-(

At least it's sad to repeat the same mistake you said dtest is doing in the test you added yourself.

Ideally, I would like to see dtest (and your test) check partial error strings, not entire English sentences.